orleans/ShoppingCart/Abstractions/Orleans.ShoppingCart.Abstractions.csproj

```
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Orleans.Sdk" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Orleans.Sdk" Version="7.*" />
```

Hold for Orleans 8.0.0-rc.2

```suggestion
    <PackageReference Include="Microsoft.Orleans.Sdk" Version="8.*" />
```